bitswap/internal/defaults/defaults.go

```diff
@@ -6,8 +6,8 @@ import (
 )
 
 const (
-	// these requests take at _least_ two minutes at the moment.
-	ProvideTimeout  = time.Minute * 3
```

Comment:
The comment was introduced at https://github.com/ipfs/boxo/commit/310fcba1dd70a0e4fa77c96a1f53b401744202e0. Probably quite outdated and the reprovider settings take care now.